### PR TITLE
add w7 follow up configurations

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/dadaab_ws_correct_dataset.json [hunk 1]
```diff
@@ -312,6 +312,17 @@
...
+    {
+      "CodeID": "code-64400270",
+      "CodeType": "Normal",
+      "DisplayText": "covid19 adaptation dadaab learning from home experience",
+      "StringValue": "covid19_adaptation_dadaab_learning_from_home_experience",
+      "NumericValue": 28,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "covid19 adaptation dadaab learning from home experience"
+         ]
```
Comment (as2388) [inline code_schemes/dadaab_ws_correct_dataset.json]: indentation

File: code_schemes/kakuma_ws_correct_dataset.json [hunk 2]
```diff
@@ -311,6 +311,17 @@
...
+    {
+      "CodeID": "code-c0c42726",
+      "CodeType": "Normal",
+      "DisplayText": "covid19 adaptation kakuma learning from home experience",
+      "StringValue": "covid19_adaptation_kakuma_learning_from_home_experience",
+      "NumericValue": 29,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "covid19 adaptation kakuma learning from home experience"
+         ]
```
Comment (as2388) [inline code_schemes/kakuma_ws_correct_dataset.json]: indentation

File: configurations/code_schemes.py [hunk 3]
```diff
@@ -41,6 +41,9 @@ class CodeSchemes(object):
     KAKUMA_HOUSEHOLD_LANGUAGE = _open_scheme("kakuma_household_language.json")
     KAKUMA_LOCATION = _open_scheme("kakuma_location.json")
 
+    KAKUMA_LEARNING_FROM_HOME_EXPERIENCE = _open_scheme('kakuma_learning_from_home_experience.json')
```
Comment (as2388) [inline configurations/code_schemes.py]: " for consistency with the surrounding lines

File: run_scripts/4_dadaab_coda_add.sh [hunk 4]
```diff
@@ -28,6 +28,8 @@ DATASETS=(
     "dadaab_s01e08"
     "dadaab_s01e09"
     "dadaab_s01e10"
+    "dadaab_learning_from_home_experience"
+
```
Comment (as2388) [inline run_scripts/4_dadaab_coda_add.sh]:
```suggestion
```

File: src/lib/pipeline_configuration.py [hunk 5]
```diff
@@ -528,7 +528,25 @@ def clean_age_with_range_filter(text):
                    raw_field_fold_strategy=FoldStrategies.assert_equal),
     ]
 
-    KAKUMA_FOLLOW_UP_SURVEY_CODING_PLANS = []
+    KAKUMA_FOLLOW_UP_SURVEY_CODING_PLANS = [
+        CodingPlan(raw_field="learning_from_home_experience_raw",
+                   dataset_name="kakuma_learning_from_home_experience_home",
```
Comment (as2388) [inline src/lib/pipeline_configuration.py]:
```suggestion
                   dataset_name="kakuma_learning_from_home_experience",
```
?

File: src/lib/pipeline_configuration.py [hunk 6]
```diff
@@ -625,7 +643,26 @@ def clean_age_with_range_filter(text):
                    raw_field_fold_strategy=FoldStrategies.assert_equal),
     ]
 
-    DADAAB_FOLLOW_UP_SURVEY_CODING_PLANS = [ ]
+    DADAAB_FOLLOW_UP_SURVEY_CODING_PLANS = [
+        CodingPlan(raw_field="learning_from_home_experience_raw",
+                   dataset_name="dadaab_learning_from_home_experience_home",
```
Comment (as2388) [inline src/lib/pipeline_configuration.py]:
```suggestion
                   dataset_name="dadaab_learning_from_home_experience",
```
?

**Review by as2388 — APPROVED** [review]

File: code_schemes/dadaab_ws_correct_dataset.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/dadaab_ws_correct_dataset.json]: updated thanks!

File: code_schemes/kakuma_ws_correct_dataset.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_ws_correct_dataset.json]: updated thanks!

File: configurations/code_schemes.py (hunk 3 above)
Comment (IsaackMwenda) [reply configurations/code_schemes.py]: updated thanks!

File: run_scripts/4_dadaab_coda_add.sh (hunk 4 above)
Comment (IsaackMwenda) [reply run_scripts/4_dadaab_coda_add.sh]: updated thanks!

File: src/lib/pipeline_configuration.py (hunk 5 above)
Comment (IsaackMwenda) [reply src/lib/pipeline_configuration.py]: updated thanks!

File: src/lib/pipeline_configuration.py (hunk 6 above)
Comment (IsaackMwenda) [reply src/lib/pipeline_configuration.py]: updated thanks!

**Pull request merged**